Esri/arcgis-maps-sdk-kotlin-samples
# Update NewModuleScript

## Description

PR to update NewModuleScript to reflect toolkit update and module name update.

## Links and Data

#3649

## What To Review

Files under `tools/NewModuleScript`

## How to Test

Try to use .jar file to create a new sample module, test and run. See [README](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tree/main/tools) for how to run the script.
